### PR TITLE
add codemods

### PR DESCRIPTION
**Pull request by kentcdodds**
**What**: This adds a codemod. It's still a work in progress

**Why**: To help people upgrade

**How**: A lot of help and inspiration from @tdeschryver

**Checklist**:

- [x] Documentation
- [x] Tests (more needed)
- [ ] Ready to be merged 
- [ ] Added myself to contributors table N/A 

We need more test cases! There are definitely code styles that this doesn't handle. We should also probably consider the `css` prop which can accept dynamic style functions as well.

### CODE REVIEW

File: other/codemods/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+To
+
+```javascript
+function(props, context) {}
+```
+
+Where `props` has a `theme` property.
+
+This codemod updates your code to use the `theme` from props and remove the
```
Comment (tdeschryver) [inline other/codemods/README.md]: A small one, but I think this should be ... and remove`s` the ... ?

Comment (kentcdodds) [inline other/codemods/README.md]: Works either way. I read it as: "updates your code ... to remove the `theme` argument"

Comment (tdeschryver) [inline other/codemods/README.md]: Oh, OK! Now I feel stupid, sorry bothering 😅

Comment (kentcdodds) [inline other/codemods/README.md]: You're no bother! You really helped get this going. Thank you!

**Pull request merged**